tutorial/getting_started/install_dependencies.md

```diff
@@ -147,7 +125,7 @@ as well and you can skip to the next section.
     jupyter notebook
     ```
 
-7. Test your napari installation. Enter the command below and an empty napari
+6. Test your napari installation. Enter the command below and an empty napari
```

Comment:
I don't think this is needed given test-env.py. The only thing they should need to test manually is launching notebook/lab.